src/cli/encrypt.ts

```diff
@@ -0,0 +1,25 @@
+let NodeRSA = require('node-rsa');
+
+export function encrypt(str: string, key: any): string {
+  if (key) {
```

Comment:
I think this condition statement is not necessary since `key` is a mandatory parameter to this function.